Assignment4/server.js

```diff
@@ -0,0 +1,9 @@
+const express = require("express");
```

Comment:
Please study and implement, how to centrally handle errors. How processes should end elegantly.Read unhandledrejection event.